dockertest/subtest.py

```diff
@@ -429,13 +471,45 @@ def try_all_stages(self, name, subsubtest):
                               self.exception_info["error_source"],
                               detail)
             exc_info = self.exception_info["exc_info"]
+            # Ignore known problems in specific docker builds
+            if self.is_known_failure(name):
+                self.final_subsubtests.add(name)
```

Comment:
I forgot if we discussed this or not...but if it's a known failure, why run the test at all?

Oh, also, what about standalone subtests (i.e. not sub-subtests), don't they get to eat cake too?

There may be a hook for the ``Subtest`` class in it's ``execute()`` method.  IIRC, that's an intermediate call by used by autotest itself to then call ``setup(), initialize(), run_once(), ...``.  Perhaps it could intercept calls to ``postprocess()``, and if ``is_known_failure()``, wrap it up into a ``try...except...`` where it can convert an actual faulure into a ``TestNAError``.  Just an idea.